policy-release/action.yml

```diff
@@ -14,57 +14,64 @@ inputs:
   GITHUB_TOKEN:
     required: true
     type: string
+  policy-working-dir:
+    description: "working directory of the policy. Useful for repos with policies in folders"
+    required: false
+    type: string
+    default: .
+  policy-version:
+    description: "release version of the policy without `v` prefix. E.g: 0.1.0"
+    required: true
+    type: string
+
 runs:
   using: "composite"
   steps:
-    -
-      name: Login to GitHub Container Registry
+    - name: Login to GitHub Container Registry
       uses: docker/login-action@v3
       with:
         registry: ghcr.io
         username: ${{ github.repository_owner }}
         password: ${{ inputs.GITHUB_TOKEN }}
-    -
-      name: Publish Wasm policy artifact to OCI registry with the 'latest' tag
+    - name: Publish Wasm policy artifact to OCI registry with the 'latest' tag
       shell: bash
       if: ${{ startsWith(github.ref, 'refs/heads/') }}
       env:
         COSIGN_EXPERIMENTAL: 1
       run: |
         set -ex
-        echo Pushing policy to OCI container registry
+        echo Pushing :latest policy to OCI container registry
         IMMUTABLE_REF=$(kwctl push -o json ${{ inputs.annotated-wasm }} ${{ inputs.oci-target }}:latest | jq -r .immutable_ref)
 
         echo Keyless signing of policy using cosign
         cosign sign --yes ${IMMUTABLE_REF}
-    -
-      name: Publish Wasm policy artifact to OCI registry with the version tag and 'latest'
+    - name: Publish Wasm policy artifact to OCI registry with the version tag and 'latest'
       shell: bash
-      if: ${{ startsWith(github.ref, 'refs/tags/') }}
+      if: ${{ ! startsWith(github.ref, 'refs/heads/') }}
       env:
         COSIGN_EXPERIMENTAL: 1
```

Comment:
As far as I can remember, we use this env to allow keyless signing. This is not necessary anymore. Therefore, we can remove this.